robosuite/models/objects/objects.py

```diff
@@ -217,12 +219,17 @@ def __init__(
             size ([float], optional): of size 1 - 3
             rgba (([float, float, float, float]), optional): Color
             density (float, optional): Density
-            friction (float, optional): tangentia friction
+            friction (float, optional): tangential friction
```

Comment:
since we're overhauling the Object classes, might be good to condense a lot of our interfaces. e.g: having both a `density` and `density_range` option seems a bit redundant and confusing; having a single argument instead that will be interpreted accordingly depending on whether the input is an iterable or single float value could be more efficient

Reply:
could also be good to either allow adjustment of all friction values, or set the torsional / rolling friction proportional to the tangential friction value

Reply:
that's true - but overhaul of Object classes should not be handled in this PR. I think we should discuss what kind of changes we will make to Object classes

Reply:
Looks like we're passing on this for now. I can tackle this in the future with you if you want after we get the v1 essentials up and running. Added this to the nice to have on our doc